sdk-core/src/main/java/care/data4life/sdk/crypto/CryptoContract.kt

```diff
@@ -61,15 +61,25 @@ class CryptoContract {
 
         fun encrypt(key: GCKey, data: ByteArray): Single<ByteArray>
         fun decrypt(key: GCKey, data: ByteArray): Single<ByteArray>
-        // TODO Change interface so it Arbitrary Data can use it directly
+
+        // ToDo move this into a convenience layer
```

Comment:
Why TODO is now ToDo?

Reply:
You are right...I miss typed it...I will change it in a minute

Reply:
done